src/services/votes.service.ts

```diff
@@ -27,3 +27,33 @@ export async function getAllVotesByFeedbackId(
     return []
   }
 }
+
+export async function toggleVote(feedbackId: number) {
+  try {
+    await axios.post(
+      `${BACKEND_API_URL}/api/votes/toggle`,
+      {
+        feedbackId,
+      },
+      {
+        headers: {
+          authorization: `Bearer ${getAccessToken()}`,
+        },
+      }
+    )
+  } catch (error) {
+    console.error(error)
+  }
+}
+
+export async function deleteVote(id: number) {
```

Comment:
If I can see it right `deleteVote` is not used anywhere. Let's remove it if it's true.